server/tweetSearch.js

```diff
@@ -4,6 +4,9 @@ module.exports = function(client) {
     var hashtags = ["#bristech", "#bristech2016"];
 
     function addTweetItem(tweets, tag) {
+        if (tweets.length === 0) {
+            return;
```

Comment:
Should this line return the tweetStore too? I can't face looking though the code to see where it's used, and it probably ignores the return value anyway but I guess consistency is good.